engine/graphics/src/graphics.h

```diff
@@ -600,17 +613,20 @@ namespace dmGraphics
      */
     void SetTextureAsync(HTexture texture, const TextureParams& paramsa);
 
-    void SetTextureParams(HTexture texture, TextureFilter minfilter, TextureFilter magfilter, TextureWrap uwrap, TextureWrap vwrap, float max_anisotropy);
-    uint32_t GetTextureResourceSize(HTexture texture);
-    uint16_t GetTextureWidth(HTexture texture);
-    uint16_t GetTextureHeight(HTexture texture);
-    uint16_t GetOriginalTextureWidth(HTexture texture);
-    uint16_t GetOriginalTextureHeight(HTexture texture);
-    void EnableTexture(HContext context, uint32_t unit, HTexture texture);
-    void DisableTexture(HContext context, uint32_t unit, HTexture texture);
-    uint32_t GetMaxTextureSize(HContext context);
-    uint16_t GetMipmapSize(uint16_t size_0, uint8_t mipmap);
-    uint8_t GetMipmapCount(uint16_t size);
+    void        SetTextureParams(HTexture texture, TextureFilter minfilter, TextureFilter magfilter, TextureWrap uwrap, TextureWrap vwrap, float max_anisotropy);
+    uint32_t    GetTextureResourceSize(HTexture texture);
+    uint16_t    GetTextureWidth(HTexture texture);
+    uint16_t    GetTextureHeight(HTexture texture);
+    uint16_t    GetOriginalTextureWidth(HTexture texture);
+    uint16_t    GetOriginalTextureHeight(HTexture texture);
+    uint32_t    GetMaxTextureSize(HContext context);
+    uint8_t     GetNumTextureHandles(HTexture texture);
+    TextureType GetTextureType(HTexture texture);
+    const char* GetTextureTypeLiteral(TextureType texture_type);
+    void        EnableTexture(HContext context, uint32_t unit, uint8_t id_index, HTexture texture);
```

Comment:
This function accepts a new argument for binding a sub-texture handle if the texture has several disparate slices